src/Tuura/Plato/Translation.hs

```diff
@@ -86,15 +86,12 @@ cartesianProduct l = removeSupersets sortAllLists
   where
     sequenced    = sequence (NonEmpty.toList l)
     removeDupes  = map nub sequenced
```

Comment:
Ah, my bad. I didn't realise this does `nub` inside each list. Sure, it should be kept then.